subscription-engine/kafka-ws/app/Kafka/KafkaTopicsConsumerCallbacksMgmt.js

```diff
@@ -114,6 +114,13 @@ class KafkaTopicsConsumerCallbacksMgmt {
     }
     return null;
   }
+
+  /**
+   * Get kafka Status for internal uses
+   */
+  getKafkaStatus() {
```

Comment:
Again, have you considered using the `getter` syntax?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/get